src/DiagnosticsProvider.php

```diff
@@ -65,6 +65,20 @@ public static function getDiagnostics($node) : \Generator {
                     }
                 }
             }
+            elseif ($node instanceof Node\Statement\NamespaceUseDeclaration) {
+                if (count($node->useClauses->children) > 1) {
+                    foreach ($node->useClauses->children as $useClause) {
+                        if($useClause instanceof Node\NamespaceUseClause && !is_null($useClause->openBrace)) {
```

Comment:
nit: backslash before built-ins functions names for consistency (PHP inlines some of it's internal functions, but requires a fully-specified name to take advantage of these optimizations from within namespaces - `is_null` in one of them, and `count` isn't, but we fully specify all internal functions so we can automatically take advantage of future optimizations)
also see https://github.com/Microsoft/tolerant-php-parser/commit/b875edcf7caa8410a4513dcf79b4add9e60831a2